### ISSUE TITLE
Logout with endsession endpoint

### ISSUE THREAD

**Issue by antoniogfs** [nextcloud/user_oidc#336]
Would it be possible to implement the logout process using the endsession endpoint as described by the OIDC standard?
Currently this is not supported.

**Comment by Mr-Markus** [comment]
There is a PR https://github.com/nextcloud/user_oidc/pull/373 from @eneiluj 
Are you still working on it and how is the status?

**Comment by eneiluj** [comment]
@Mr-Markus Yes it's still a WIP. It's coming soon though as I feel the implementation is mostly done. I just have some doubts I want to clarify.

**Comment by joshtrichards** [comment]
Added in #373
